app/views/categories/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<!-- app/views/categories/index.html.erb -->
+
+<h2>Categories</h2>
+
+<% @categories.each do |category| %>
+  <div>
+  <img src="<%= category.icon %>" />
+    <h2><%= category.name %></h2>
+
+    <!-- calculate total amount spent on this category -->
+    <p>Total Spent: $<%= category.deals.sum(:amount) %></p>
+
```

Comment:
- [ ] You did a great job on the project functionalities but you didn't add any styles that follows the given design. Please make sure to add styles to your project and it should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
  - Colors.
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.
 
Adding styles will improve the User experience visiting your application.

### Your design

![image](https://github.com/Abdulwhbhussain/Budget-app/assets/83803414/a6259c6c-07c1-41c2-a0b2-d31b0e938e06)

![image](https://github.com/Abdulwhbhussain/Budget-app/assets/83803414/74557c39-a96d-4b9d-8d96-33bbc71aa00b)

![image](https://github.com/Abdulwhbhussain/Budget-app/assets/83803414/1d5a9b31-0da0-4196-97f0-97d09fce8f08)

![image](https://github.com/Abdulwhbhussain/Budget-app/assets/83803414/b3c980a6-5957-4b9b-bef5-e2101cde7e27)


### Given design

![image](https://github.com/Abdulwhbhussain/Budget-app/assets/83803414/7c789b29-e655-4d07-a301-2f240eedebc5)

![image](https://github.com/Abdulwhbhussain/Budget-app/assets/83803414/ababd4ab-4427-4404-bd6d-fc9b104caa14)

Reply:
I improved the design!